Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -647,18 +626,15 @@ if (!($noDiscoveryService)) {
     Write-Host ""
     Write-Host "Registering with Discovery Service."
     Write-Host ""
-    Add-DiscoveryRegistration $discoveryServiceUrl "$authorizationServiceUrl/v1" $credential
+    $buildVersion = [System.Diagnostics.FileVersionInfo]::GetVersionInfo("$appDirectory\Fabric.Authorization.API.dll").FileVersion
+    $serviceName = "AuthorizationService"
+    $serviceVersion = 1
+    $friendlyName = "Fabric.Authorization"
+    $description = "The Fabric.Authorization service provides centralized authorization across the Fabric ecosystem."
+    Add-DiscoveryRegistration $discoveryServiceUrl "$authorizationServiceUrl/v1" $credential, $buildVersion, $serviceName, $serviceVersion, $friendlyName, $description
```

Comment:
Same comment as in the other repo, it might be better to encapsulate the body of the post into its own object so we can reduce the number of parameters that need to be passed to this function.